packages/@uppy/dashboard/src/utils/getFileTypeIcon.js

```diff
@@ -115,7 +115,7 @@ module.exports = function getIconByMime (fileType) {
   }
 
   // Archive
-  const archiveTypes = ['zip', 'x-7z-compressed', 'x-rar-compressed', 'x-gtar', 'x-apple-diskimage', 'x-diskcopy']
```

Comment:
Could you clarifty why some of those are removed?

Reply:
I have replaced `x-gtar` ".gtar" which is rarely used with `x-tar` & `x-gzip` ".tar" ".gz" which are more common. While for the `x-diskcopy` i didn't find any valid mime type with this subtype